bootstrap_form.gemspec

```diff
@@ -1,8 +1,7 @@
-lib = File.expand_path("../lib", __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+$LOAD_PATH.push File.expand_path("lib", __dir__)
```

Comment:
Dependency declarations aside (if you want to move them to the Gemfile, no problem), and with the exception of the addition of the `required_ruby_version` (nice catch, thanks), I would prefer the other gemspec changes be reverted unless you have a strong objection.

My overall reasoning is that the existing gemspec is based on the template provided by `bundle gem`, which presumably has many eyes on it has been well vetted by the community. It has also served me well on many gems I work on.

More specifically, according to the [official docs](https://github.com/rubygems/guides/blob/543ae72b78678d4f9b978f7070635d553687ec38/specification-reference.md):

* `platform` should be left as the default value "unless the code will only compile on a certain type of system"
* `files` should not include tests (not a strict requirement, but it is common practice to exclude them)
* `bindir = "exe"` is a good safety measure to ensure binstubs or other helper scripts that we may add to `bin/` in the future are not inadvertently packaged with the gem

Reply:
Ah neat. I just used whatever `rails plugin new` would produce with some tweaks. I reverted gemspec to more or less original state minus dev dependencies. I still don't undertand load_path checking is for, but it doesn't hurt I suppose.

Reply:
Gotcha on `gem install bundler --no-document`